.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist/
```

Comment:
i dont see changes to the action manifest. Shouldnt that point to dist/index.js now?

Reply:
make sure e2e still works